src/components/NewMovie/NewMovie.tsx

```diff
@@ -37,12 +35,15 @@ export const NewMovie: React.FC<Props> = ({ onAdd }) => {
 
     onAdd(newMovie);
     setCount(count + 1);
-    clear();
+    clearForm();
   };
 
-  const isBouttonActive = !(
-    title && imgUrl && imdbUrl && imdbId
-  );
+  const isSubmitDisabled = [
+    title,
+    imgUrl,
+    imdbUrl,
+    imdbId,
+  ].every(field => field.trim().length > 0) && imgUrl && imdbUrl && imdbId;
```

Comment:
```suggestion
  ].every(field => field.trim().length > 0);
```

Useless chained checks